src/shapes.ts

```diff
@@ -91,6 +91,16 @@ export interface OpenFinWindow {
     id: number;
     name: string;
     uuid: string;
+    preloadState?: { // set if window has a preload script
+        src: String;
```

Comment:
do we really need `src` anyway? will `src` ever be anything other than the preload option? If not, isn't the option always avaiable via [`getOptions`](https://github.com/HadoukenIO/core/blob/develop/src/browser/api/window.js#L1062)?